modules/@apostrophecms/i18n/index.js

```diff
@@ -1,38 +1,38 @@
-// TODO: replace i18n npm module with i18n-next. Careful, the middleware for apostrophe editing
-// locales is also in here.
+// This module makes an instance of the [i18next](https://npmjs.org/package/i18next) npm module available
+// in Nunjucks templates via the `__t()` helper function. That function is also available on `req` objects
+// as `req.t()`. Any options passed to this module are passed on to `i18next`.
 //
-// This module makes an instance of the [i18n](https://npmjs.org/package/i18n) npm module available
-// in Nunjucks templates via the usual `__()` helper function. That function and its relatives are
-// also available on `req` objects. Any options passed to this module are passed on to `i18n`.
-//
-// `apos.i18n.i18n` can be used to directly access the `i18n` npm module instance if necessary.
-// It usually is a bad idea. Use `req.__` if you need to localize in a route.
-//
-// By default i18n locale files are generated in the `locales` subdirectory of the project.
-//
-// ## Options
-//
-// `localesDir`: if specified, the locale `.json` files are stored here, otherwise they
-// are stored in the `locales` subdirectory of the project root.
+// `apos.i18n.i18next` can be used to directly access the `i18next` npm module instance if necessary.
+// It usually is not necessary. Use `req.t` if you need to localize in a route.
 
-const _ = require('lodash');
-const i18n = require('i18n');
+const i18next = require('i18next');
+const i18nextHttpMiddleware = require('i18next-http-middleware');
+const fs = require('fs');
 
 module.exports = {
   options: {
-    alias: 'i18n'
+    alias: 'i18n',
+    i18n: {
```

Comment:
Opting in to having some localizations of our own like any other module might.

Reply:
I can't tell if this is for demonstration purposes given that no other module with localized stings has the option. At the moment I'm pretty thoroughly confused how this works or if our strings would fall over if a project set a different namespace here.

Reply:
Any module may have an `i18n` option, which may specify any namespace because for translation team management purposes it often makes sense not to artificially require three related modules to have their own translations of the same phrase. In this case I chose to put them all together in the `i18n` module because it was convenient but I could have fanned these out over different modules if I wanted to.

It's not for demonstration purposes per se, it's because this is how we do it and the `@apostrophecms/i18n` module does it like any other would.